widgets/text/text.go

```diff
@@ -109,6 +109,17 @@ func (t *Text) Write(text string, wOpts ...WriteOption) error {
 		t.reset()
 	}
 	for _, r := range text {
+		// Implement basic buffer limit to avoid excess memory issues
+		if len(t.content) > 1 && len(t.content) >= t.opts.maxContent && t.opts.maxContent > 0 {
```

Comment:
Can you help me understand the first condition `len(t.content) > 1`?

Why do we need it, or in other words wouldn't the rest of the condition be enough? I.e.

if len(t.content) >= t.opts.maxContent && t.opts.maxContent > 0 { ... }

Reply:
I've fixed this, you are correct in that it isn't required.